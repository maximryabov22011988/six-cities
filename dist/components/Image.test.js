'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Image_1 = require('./Image');
it('Image correctly render', () => {
  const tree = renderer
    .create(React.createElement(Image_1.default, { src: '/img', height: '100', width: '100' }))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Image.test.js.map
