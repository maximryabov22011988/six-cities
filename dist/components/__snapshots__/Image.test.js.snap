// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image correctly render 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "#e1e1e1",
      "borderRadius": "4px",
      "display": "flex",
      "height": "100px",
      "justifyContent": "center",
      "width": "100px",
    }
  }
>
  <img
    height={0}
    onError={[Function]}
    onLoad={[Function]}
    src="/img"
    width={0}
  />
  <span>
    Loading ...
  </span>
</div>
`;
